### ISSUE TITLE
Reposts without a Twitter syndication are POSSEd to Twitter as HTML

### ISSUE THREAD

**Issue by Lancey6** [kylewm/redwind#72]
If you repost a note without a Twitter syndication link and share it to Twitter, the post's raw HTML is tweeted. Additionally, any images will not be included in the tweet.

For example, this post: http://lancey.space/2015/08/repost-of-kyle-mahan-literally-united-is-pleased
Repost of this post: https://kylewm.com/2015/08/literally-united-is-pleased-to-offer-me-to-play-games
The original post does not have a Twitter syndication link (but does have an instagram synd link). Attempting to share the post on Twitter gets this as the preview:

```
Kyle Mahan: <p>Literally, United is pleased to offer me to play games on my phone</p>
                
        <a… https://kylewm.com/2015/08/literally-united-is-pleased-to-offer-me-to-play-games
```

**Comment by Lancey6** [comment]
Confirming that this is true of pure text reposts as well.

This post: http://lancey.space/drafts/0b3cb11f62d9da391a96b95f134a2653
Is a repost of this post: https://kylewm.com/2015/08/oh-interesting-thank-you-honestly-my-issues-are-all
Which doesn't have a Twitter syndication link. Sharing it gets this as the preview:

```
Kyle Mahan: <p>Oh interesting, thank you! Honestly my issues are all preflight. Once we’re in the air I’m usually… https://kylewm.com/2015/08/oh-interesting-thank-you-honestly-my-issues-are-all
```

**Comment by Lancey6** [comment]
Corrected in abd6bec7ce4a7dd3879aed171519e6ef529b7215, but I'm going to also work on reposted images sharing to Twitter.
